src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageSolutionDetailControlModel.cs

```diff
@@ -80,6 +80,7 @@ private void UpdateInstalledVersions()
                     else
                     {
                         project.InstalledVersion = null;
+                        project.AutoReferenced = false;
```

Comment:
Is this problem not there in the Project specific PM UI?

Reply:
@nkolev92 yes, it only repros on the solution level UI